docroot/modules/custom/foia_webform/src/Plugin/WebformHandler/FoiaSubmissionQueueHandler.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Drupal\foia_webform\Plugin\WebformHandler;
+
+use Drupal\webform\Plugin\WebformHandler\EmailWebformHandler;
+use Drupal\webform\WebformSubmissionInterface;
+
+/**
+ * Emails a webform submission.
+ *
+ * @WebformHandler(
+ *   id = "foia_submission_queue",
+ *   label = @Translation("FOIA Submission Queue"),
+ *   category = @Translation("Queueing"),
+ *   description = @Translation("Queues a webform submission to be sent later."),
+ *   cardinality = \Drupal\webform\WebformHandlerInterface::CARDINALITY_UNLIMITED,
+ *   results = \Drupal\webform\WebformHandlerInterface::RESULTS_PROCESSED,
+ * )
+ */
+class FoiaSubmissionQueueHandler extends EmailWebformHandler {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function postSave(WebformSubmissionInterface $webform_submission, $update = TRUE) {
+    $state = $webform_submission->getWebform()->getSetting('results_disabled') ? WebformSubmissionInterface::STATE_COMPLETED : $webform_submission->getState();
+    if (in_array($state, $this->configuration['states'])) {
+      // Get the Webform generated email message.
+      $message = $this->getMessage($webform_submission);
+      // Add the submission to the queue.
+      $this->queueSubmission($webform_submission, $message);
+    }
+  }
+
+  /**
+   * Adds the submission to the foia_form_manual_submitter queue.
```

Comment:
This comment needs to be cleaned up to reference the right queue name. Or you can keep it generic and say that the submission is added to a queue - whatever your prefer!